job_controller/job_controller.go

```diff
@@ -72,7 +79,7 @@ type ControllerInterface interface {
 	CreateService(job interface{}, service *v1.Service) error
 
 	// DeleteService deletes the service
-	DeleteService(job interface{}, service *v1.Service) error
+	DeleteService(job interface{}, name string, namespace string) error
```

Comment:
Can we make function signature consistent with CreateService?

Reply:
Personally, I think we do not need to compose a service object to delete it.

Reply:
Is service argument meant to have the entire Service Object?  What is the `job` argument for?

Reply:
the job argument is for the purpose of logging with job contexts such as its name
e.g  the r.Recorder.Eventf takes in the job as the argument